scripts/sem_bleaching_map.py

```diff
@@ -311,137 +349,315 @@ def configure_sem_for_tile(self):
             sem_pxs = self.ebeam.pixelSize.value
             scale = (self.subpxs / sem_pxs[0], self.subpxs / sem_pxs[1])
             self.ebeam.scale.value = scale
-        self.ebeam.resolution.value = self.tile_shape # just a spot
+        self.ebeam.resolution.value = self.tile_shape  # just a spot
 
-    def calc_xy_pos(self, roi, pxs):
+    def configure_sem_for_survey(self, roi):
         """
-        Compute the X and Y positions of the ebeam
+        Configure the SEM for regular survey image
+        """
+        self.ebeam.dwellTime.value = self.dtsem
+        self.ebeam.scale.value = (1, 1)
+
+        sem_pxs = self.ebeam.pixelSize.value
+        ccd_pxs = self.get_ccd_pxs()
+
+        full_width = self.ebeam.resolution.range[1]
+
+        # calculate the width of the border in rel coordinates
+        b_rel = [BORDER * ccd_pxs[0] / (full_width[0] * sem_pxs[0]),
+                 BORDER * ccd_pxs[1] / (full_width[1] * sem_pxs[1])]
+
+        self.ccd.fanSpeed.value = 0
+        self.ccd.targetTemperature = 25
+        time.sleep(0.5)  # wait half a second for vibrations to stop
+
+        rel_width = [(roi[2] - roi[0]) + 2.0 * b_rel[0], (roi[3] - roi[1]) + 2.0 * b_rel[1]]
+        rel_center = [(roi[0] + roi[2]) / 2, (roi[1] + roi[3]) / 2]
+
+        trans = [full_width[0] * (rel_center[0] - 0.5) - self.drift[0],
+                 full_width[1] * (rel_center[1] - 0.5) - self.drift[1]]
+
+        resolution = [int(full_width[0] * rel_width[0]),
+                      int(full_width[1] * rel_width[1])]
+
+        # configure the ebeam location
+        self.ebeam.translation.value = trans  # px (0 = center)
+        self.ebeam.resolution.value = self.ebeam.resolution.clip(resolution)
+
+    def configure_and_acquire_sem_survey(self, roi):
```

Comment:
Let's just call this "acquire_sem_survey". It should be a given that the SEM gets configured.